packages/nodes/mention/src/withMention.spec.tsx

```diff
@@ -212,6 +212,27 @@ describe('withMention', () => {
         focus: { path: [0, 0], offset: 1 },
       });
     });
+
+    it('should remove the input when inserting break in empty input', () => {
```

Comment:
I don't understand the empty input special case. Also, i believe the action on insertBreak should be a no-op, it should not remove the mention. We have escape to remove the mention, enter should do nothing, regardless of whether it's empty or not.